src/Http/Http.Extensions/gen/RequestDelegateGeneratorSources.cs

```diff
@@ -388,14 +388,20 @@ public override bool HasDefaultValue
 
         public override object[] GetCustomAttributes(Type attributeType, bool inherit)
         {
-            var attributes = _constructionParameterInfo?.GetCustomAttributes(attributeType, inherit);
+            var constructorAttributes = _constructionParameterInfo?.GetCustomAttributes(attributeType, inherit);
 
-            if (attributes == null || attributes is { Length: 0 })
+            if (constructorAttributes == null || constructorAttributes is { Length: 0 })
             {
-                attributes = _underlyingProperty.GetCustomAttributes(attributeType, inherit);
+                return _underlyingProperty.GetCustomAttributes(attributeType, inherit);
             }
 
-            return attributes;
+            var propertyAttributes = _underlyingProperty.GetCustomAttributes(inherit);
```

Comment:
```suggestion
            var propertyAttributes = _underlyingProperty.GetCustomAttributes(attributeType, inherit);
```

Reply:
Good catch! We don't ever call the overload that takes a type since we filter for the attributes we want _after_ calling `GetCustomAttributes`.